app/main.py

```diff
@@ -1 +1,9 @@
-# write your code here
+def copy_file(command: str) -> None:
+    files_names = command.split()
+    if files_names[1] == files_names[2]:
+        return
+
+    with (open(files_names[1], "r") as main_file,
+          open(files_names[2], "w") as copied_file):
+        for line in main_file:
+            copied_file.write(line)
```

Comment:
Do not iterate through lines here, you could use `.writelines()` or a combination of `.write()` + `.read()` methods.